packages/studio/tests/utils/removeTopLevelFragments.test.ts

```diff
@@ -50,41 +48,3 @@ it("removes top level fragments from a PageState record", () => {
     },
   ]);
 });
-
-it("removes top level fragments from a FileMetadata record", () => {
```

Comment:
It looks like `removeTopLevelFragments` is only useful for things that have a `ComponentTree`.  Before, `FileMetadata` included `moduleMetadata` which had a `ComponentTree`.  Now, `FileMetadata` only includes `ComponentMetadata` and `ErrorFileMetadata`,  neither of which have a component tree.  Does it still make sense to call `removeTopLevelFragments` on `ComponentMetadata`?  I see we still do that in several places, and I'm not sure if it's useful anymore.